DQMServices/Core/src/DQMStore.cc

```diff
@@ -405,14 +405,14 @@ void DQMStore::mergeAndResetMEsRunSummaryCache(uint32_t run,
 	{
 	  if(me->getTH1()->CanExtendAllAxes() && i->getTH1()->CanExtendAllAxes()) {
 	    TList list;
-	    list.Add(i->getTH1());
+	    list.Add(i->getTH1().operator->().operator->());
```

Comment:
I did mean the lock could be released before the call to `Add()` since the lifetime of the temporary returned from `getTH1` is not guaranteed to last during the entire line.